src/lwc/rd2ElevateInformation/rd2ElevateInformation.js

```diff
@@ -390,6 +526,14 @@ export default class rd2ElevateInformation extends NavigationMixin(LightningElem
         return `text ${this.fields.commitmentId.label}`;
     }
 
+    get qaLocatorLastFourDigits() {
+        return `text Last Four Digits`;
+    }
+
+    get qaLocatorExpirationDate() {
+        return `text Expiration Date`;
+    }
+
     get qaLocatorNewWindow() {
         return `link ${this.labels.textNewWindow}`;
```

Comment:
I'll be honest and say that I do not like this pattern of using custom labels for locators. Locator names should be static and never change. However, by using a custom label for the locator, the value can change completely outside of the context of the component, the jest test, or even robot tests. Even localization can break all of these things, and I believe there has been work in getting robot to run in other languages.

Reply:
I talk with Ratna and she prefers the hard-coded value as well